integration-tests/test/native-eth-ovm-calls.spec.ts

```diff
@@ -32,39 +36,57 @@ describe('Native ETH value integration tests', () => {
       ]
     }
 
-    const checkBalances = async (
-      expectedBalances: BigNumber[]
-    ): Promise<void> => {
-      const realBalances = await getBalances()
-      expect(realBalances[0]).to.deep.eq(expectedBalances[0])
-      expect(realBalances[1]).to.deep.eq(expectedBalances[1])
+    const expectBalancesWithinRange = (
```

Comment:
Why is the range required here? Is it because of the gas usage?

Reply:
Yes, because of the gas usage. I can't easily assert the exact balance because something somewhere is spending a small amount (~46k) of Wei outside the test.